src/catchup/VerifyLedgerChainWork.cpp

```diff
@@ -94,46 +95,48 @@ VerifyLedgerChainWork::getStatus() const
     {
         std::string task = "verifying checkpoint";
         return fmtProgress(mApp, task, mRange.first(), mRange.last(),
-                           mCurrCheckpoint);
+                           (mRange.last() - mCurrCheckpoint));
     }
     return Work::getStatus();
 }
 
 void
 VerifyLedgerChainWork::onReset()
 {
-    auto setLedger = mApp.getLedgerManager().getLastClosedLedgerHeader();
-    if (setLedger.header.ledgerSeq < 2)
-    {
-        setLedger = {};
-    }
-    if (mFirstVerified.header.ledgerSeq != 0)
-    {
-        mFirstVerified = setLedger;
-    }
-    if (mLastVerified.header.ledgerSeq != 0)
+    // Note that mFirstVerified is not reset here, since it is set
```

Comment:
Instead of this comment, I think it would be much cleaner to always reset `mFirstVerified` to the zero value.